src/Features/Core/Portable/UseNameOf/AbstractUseNameOfDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace Microsoft.CodeAnalysis.UseNameof
+{
+    using System.Collections.Immutable;
+    using System.Linq;
+    using System.Threading;
+    using Microsoft.CodeAnalysis.CodeStyle;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.CodeAnalysis.Editing;
+    using Microsoft.CodeAnalysis.LanguageServices;
+    using Microsoft.CodeAnalysis.Operations;
+
+    internal abstract class AbstractUseNameOfDiagnosticAnalyzer : AbstractCodeStyleDiagnosticAnalyzer
+    {
+        protected AbstractUseNameOfDiagnosticAnalyzer()
+            : base(
+                IDEDiagnosticIds.UseNameofDiagnosticId,
+                "Use nameof.",
+                "Use nameof.",
```

Comment:
needs loc.

Reply:
No idea how, help needed here, no rush.

Reply:
take a look at our other analyzers in this project :)